sml_builder/help_centre.py

```diff
@@ -72,10 +126,7 @@ def guidances(category, sub_category=None):
         escaped_text = escape(text)
         body = Markup(markdown.markdown(escaped_text))
     else:
-        body = False
-
-    help_centre_nav = _help_centre_nav(category)
-
+        body = None
```

Comment:
The code linters flag this since the the 'body' variable is technically changing type to a boolean, this is so we can check {% if body %} in the html file. 'body = None' achieves the same thing without the linters flagging it.